registration-processor/workflow-engine/registration-processor-workflow-manager-service/src/main/resources/bootstrap.properties

```diff
@@ -1,17 +1,23 @@
 spring.cloud.config.uri=http://localhost:51000/config
 spring.cloud.config.label=master
 spring.application.name=application,registration-processor
-spring.profiles.active=mz
+spring.profiles.active=default
 spring.cloud.config.name=registration-processor
 management.endpoints.web.exposure.include=refresh
 registration.processor.zone=secure
 vertx.cluster.configuration=${spring.cloud.config.uri}/*/${spring.profiles.active}/${spring.cloud.config.label}/hazelcast_${registration.processor.zone}.xml
 #management.security.enabled=false
 #disabling health check so that client doesnt try to load properties from sprint config server every
 # 5 minutes (should not be done in production)
+#MessageSender api id
+mosip.regproc.user.notification.api.id=mosip.registration.processor.user.notification
```

Comment:
Please remove mosip.regproc.user.notification.api.id and version from bootstrap and add in mosip-config

Reply:
done ,please check earliest commit